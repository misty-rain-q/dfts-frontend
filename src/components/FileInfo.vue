<template>
    <div class="fileInfo">
        <p>剩余下载时间：</p>
        <p>剩余下载次数：</p>
        <p>上传时间：</p>
    </div>
</template>

<style scoped>
.fileInfo {
    text-align: left;
    margin-top: 20px;
}
</style>
