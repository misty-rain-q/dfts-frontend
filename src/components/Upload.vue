<template>
  <div class="upload">
    <a-button @click="turnToUploadPage" type="primary" icon="upload" :size="size">
      上传
    </a-button>
  </div>
</template>

<script>
// @ is an alias to /src
// import HelloWorld from '@/components/HelloWorld.vue';

export default {
  name: 'Upload',
  components: {
  },
  methods: {
    turnToUploadPage() {
      console.log('UploadPage');
    },
  },
};
</script>

<style scoped>

</style>
