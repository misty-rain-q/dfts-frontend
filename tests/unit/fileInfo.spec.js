import { mount } from '@vue/test-utils';
import FileInfo from '@/components/FileInfo.vue';
import Vue from 'vue';
import Button from 'ant-design-vue/lib/button';
import Modal from 'ant-design-vue/lib/modal';
import Input from 'ant-design-vue/lib/input';
import CheckBox from 'ant-design-vue/lib/checkbox';
import Table from 'ant-design-vue/lib/table';
import Drawer from 'ant-design-vue/lib/drawer';
import VideoPlayer from 'vue-video-player';
import Viewer from 'v-viewer';
import Upload from 'ant-design-vue/lib/upload';
import Select from 'ant-design-vue/lib/select';
import InputNumber from 'ant-design-vue/lib/input-number';

// Vue.use(ant);
Vue.use(Table);
Vue.use(Drawer);
Vue.use(Viewer);
Vue.use(VideoPlayer);
Vue.use(Button);
Vue.use(Modal);
Vue.use(Input);
Vue.use(CheckBox);
Vue.use(Upload);
Vue.use(Select);
Vue.use(InputNumber);

const wrapper = mount(FileInfo);

describe('FileInfo.vue', () => {
  it('matches snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
