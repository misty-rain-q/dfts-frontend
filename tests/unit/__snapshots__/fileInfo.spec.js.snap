// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FileInfo.vue matches snapshot 1`] = `
<div class="fileInfo">
  <p>剩余下载时间：</p>
  <p>剩余下载次数：</p>
  <p>上传时间：</p>
</div>
`;
